src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -84,6 +87,11 @@ class ConfigurationProvider implements ConfigurationProviderInterface
      */
     protected const DEFAULT_REPORTING_ENABLED = false;
 
+    /**
+     * @var int
+     */
+    protected const DEFAULT_MANIFESTS_RATING_THRESHOLD = 0;
```

Comment:
afaik it should be 70%, isn't it?

Reply:
should be 0 to execute all the manifests